src/jit/codegencommon.cpp

```diff
@@ -8687,8 +8687,10 @@ void CodeGen::genFnProlog()
     {
         excludeMask |= RBM_PINVOKE_FRAME;
 
+#ifndef _ARM_
         assert((!compiler->opts.ShouldUsePInvokeHelpers()) || (compiler->info.compLvFrameListRoot == BAD_VAR_NUM));
```

Comment:
This does not look right. `compLvFrameListRoot` should not be allocated at all with ShouldUsePInvokeHelpers.